integration-tests/integration/login/login.spec.js

```diff
@@ -53,7 +52,8 @@ context('Login functionality', () => {
   it('Log in as ordinary user receives unauthorised', () => {
     cy.task('stubLogin', { username: 'joe', roles: [{}] })
     cy.login()
-    UnauthorisedPage.verifyOnPage()
+    const menuPage = MenuPage.verifyOnPage()
+    menuPage.noAdminFunctionsMessage().contains('There are no admin functions associated with your account.')
```

Comment:
this was previously a bit naff, but now use the menu page to serve the unauthorised instead